force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -271,13 +313,20 @@ public with sharing class FlowControlLoopsCollections {
      */
     public static Integer findWordInList(String wordToFind, List<String> words) {
         // The variable to store the index of the word
-
-        // Loop through the list of words
-            // If the current word is the word to find, exit the loop
-
-        return null; // Replace null with the variable you used to store the result
+        Integer wordPosition;
+        // Check each word one by one to see if its the one we're looking for
+        for (Integer i = 0; i < words.size(); i++) {
+             // If the current word is the word we want, exit the loop
+            if (words.get(i) == wordToFind) {
+                wordPosition = i;
+                break;
+            }
+        wordPosition = -1;
+        }
+        return wordPosition;  // Passes Debug
     }
```

Comment:
Good job.  I would  rather exit the loop as soon as I find a match. You could still be iterating over 1000s of values even though you  found what you wanted. 

```
 public static Integer findWordInList(String wordToFind, List<String> words) {
     for(Integer i = 0; i < words.size(); i++){
         if(words[i] == wordToFind){
             return i;
         }
     }
        return -1; // Replace null with the variable you used to store the resulto
    }
    ```